docs/website/blog/2022-12-13-odo-v3.4.0.md

```diff
@@ -0,0 +1,172 @@
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/fWz7UPOgJdA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+#### Listing Devfile Stack versions
+
+[`odo registry`](../docs/command-reference/registry) (and its [JSON counterpart](../docs/command-reference/json-output#odo-registry--o-json)) now displays the Devfile stack versions.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/DFmWsJ75JNM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Detailed Changelog
+
+As with every release, you can find the full list of changes and bug fixes on the [GitHub release page](https://github.com/redhat-developer/odo/releases/tag/v3.4.0).
+
+**Features/Enhancements:**
+
+- Automate release process \(GitHub part\) [\#6403](https://github.com/redhat-developer/odo/pull/6403) ([rm3l](https://github.com/rm3l))
+- Set Priority field in GitHub Project item based on the 'priority/\*' label on the issue [\#6400](https://github.com/redhat-developer/odo/pull/6400) ([rm3l](https://github.com/rm3l))
+- Show Devfile Stack versions in "odo registry" output [\#6397](https://github.com/redhat-developer/odo/pull/6397) ([rm3l](https://github.com/rm3l))
+- Add complete devfiles to advanced guides [\#6388](https://github.com/redhat-developer/odo/pull/6388) ([valaparthvi](https://github.com/valaparthvi))
+- Pass caller to registry telemetry [\#6382](https://github.com/redhat-developer/odo/pull/6382) ([feloy](https://github.com/feloy))
+- Get next free port when forwarding ports on podman [\#6377](https://github.com/redhat-developer/odo/pull/6377) ([feloy](https://github.com/feloy))
+- Add support for parsing multiple k8s definition in a single Devfile K8s component [\#6372](https://github.com/redhat-developer/odo/pull/6372) ([valaparthvi](https://github.com/valaparthvi))
+- odo list component shows components running on podman [\#6366](https://github.com/redhat-developer/odo/pull/6366) ([feloy](https://github.com/feloy))
+- Support Alizer-based automatic port detection with odo init interactive mode [\#6365](https://github.com/redhat-developer/odo/pull/6365) ([rm3l](https://github.com/rm3l))
+- Update build version and release blog post for v3.3.0 [\#6359](https://github.com/redhat-developer/odo/pull/6359) ([valaparthvi](https://github.com/valaparthvi))
+- Sync files on Podman [\#6348](https://github.com/redhat-developer/odo/pull/6348) ([feloy](https://github.com/feloy))
+- Set labels for component on podman [\#6329](https://github.com/redhat-developer/odo/pull/6329) ([feloy](https://github.com/feloy))
+- odo dev on podman: Add support for devfile volume [\#6328](https://github.com/redhat-developer/odo/pull/6328) ([feloy](https://github.com/feloy))
```

Comment:
I think some of these entries could be dispatched in other entries, to only keep features/enhancements for end-users.

Reply:
Done - thanks.
See [`bf27935` (#6407)](https://github.com/redhat-developer/odo/pull/6407/commits/bf27935c935d0c48fc51af7869f75caeb38895a1)